### DISCUSSION TITLE
The next release (6.2.64) and betas (6.2.63b) of Sympa

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1623] [Release works]
Hi @everyone.

Due date of the next stable (6.2.64) is proposed to be 21st Jun. 2021.
Maybe the first beta (6.2.63b.1) will come out in May.

See also [the milestone 6.2.64](https://github.com/sympa-community/sympa/milestone/17).

**Comment by ikedas** [comment]
I propose 6.2.63b.1 will be released in 31 May.

Currently only undone work in [milestone 6.2.64](https://github.com/sympa-community/sympa/milestone/17) is #466.

**Comment by ikedas** [comment]
Done!

**Comment by racke** [comment]
Thanks a lot!

**Comment by ikedas** [comment]
I think 6.2.64 may be postponed for a week or so. We'd be better to check carefully the recent fixes including sympa-community/sympa#1196 .
  - 6.2.63b.2 may be out around 21st Jun.
  - 6.2.64 may be released around the end of June.

---
N.B. #1196 above updates dependencies (see cpanfile).

**Comment by ikedas** [comment]
Done!

**Comment by ikedas** [comment]
Done!

**Comment by racke** [comment]
Excellent, thanks!
